test-case-reporting/src/setup_db.ts

```diff
@@ -25,6 +25,8 @@ export async function setupDb(db: Database): Promise<unknown> {
   return db.schema
     .createTable('builds', table => {
       table.increments('id').primary();
+      // table.string('commit_sha', 40) makes a varchar, which is not necessary;
+      // commit shas will always be 40 characters long.
```

Comment:
If this isn't indexed, let's leave it as a string

Reply:
Done.